package.json

```diff
@@ -8,7 +8,9 @@
   "author": "Eli Perelman <eli@eliperelman.com>",
   "license": "MPL-2.0",
   "scripts": {
-    "bootstrap": "oao bootstrap",
+    "fix-line-endings": "find packages -name package.json | xargs sed -i '' -e '$a\\'",
```

Comment:
This line is temporary until oao fixes missing line endings in re-writing `package.json`s.